ElementaryTasks/Program.cs

```diff
@@ -37,6 +37,10 @@ static void Main(string[] args)
                 75 - 89   B
                 90 - 100  A*/
             //logicalOperations.StudentEvaluation();
+
+            var cicles = new Cicles();
+
+            cicles.SumOfEvenValues();
```

Comment:
No task provided. What should be done here?)

Reply:
fixed)